spring-integration-jmx/src/test/java/org/springframework/integration/jmx/MBeanAttributeFilterTests.java

```diff
@@ -105,7 +105,7 @@ public void testAttributeFilterNot() {
 		Map<String, Object> bean = (Map<String, Object>) payload
 				.get(domain + ":name=in,type=MessageChannel");
 
-		assertEquals(8, bean.size());
+		assertEquals(9, bean.size());
```

Comment:
It isn't obvious for normal people, so let's add all those 9 attributes to test!